dpppt-backend-sdk/dpppt-backend-sdk-ws/src/main/java/org/dpppt/backend/sdk/ws/controller/DPPPTController.java

```diff
@@ -193,6 +199,9 @@ public DPPPTController(DPPPTDataService dataService, String appSource,
 	@GetMapping(value = "/buckets/{dayDateStr}", produces = "application/json")
 	public @ResponseBody ResponseEntity<BucketList> getListOfBuckets(@PathVariable String dayDateStr) {
 		OffsetDateTime day = LocalDate.parse(dayDateStr).atStartOfDay().atOffset(ZoneOffset.UTC);
+		if (day.toEpochSecond() < 0) {
```

Comment:
Why `< 0`? I don't see any problems in being before 1970 (except for it being an unrealistic date per se)... One could introduce a check to see if the date is older than 21 days (retention period) or something similar.

Reply:
Dates with large negative years (e.g. `v1/buckets/-900000000-01-01`) are parsed by the controller and can trigger a long overflow exception which is reported in the response. Since this is not a realistic use case and I can't think of a real security concern here, I removed the check for better readability.